iXBRLViewerPlugin/__init__.py

```diff
@@ -203,10 +205,13 @@ def iXBRLViewerSaveCommand(cntlr):
     dialog = SaveViewerDialog(cntlr)
     dialog.render()
     if dialog.accepted and dialog.filename():
+        scriptUrl = dialog.scriptUrl()
+        if scriptUrl == DEFAULT_JS_FILENAME:
+            scriptUrl = DEFAULT_VIEWER_PATH
```

Comment:
This might be confusing if a user actually wants a relative URL to `ixbrlviewer.js`.  Would it be better to have a separate "use default" checkbox, or use empty string to get the default with a "leave blank for default" message.

Reply:
I wasn't expecting users to enter relative paths since it isn't clear from the GUI where a relative path is starting from, but I went ahead and made it a blank string.